packages/react-core/src/components/ProgressStepper/examples/ProgressStepperIcons.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { ProgressStepper, ProgressStep } from '@patternfly/react-core';
+import InProgressIcon from '@patternfly/react-icons/dist/esm/icons/in-progress-icon';
+import PendingIcon from '@patternfly/react-icons/dist/esm/icons/pending-icon';
+
+<ProgressStepper>
```

Comment:
As a nitpick in addition to adding the `export...`, naming the file/export as "ProgressStepperCustomIcons" is slightly more fitting